.github/workflows/deploy.yml

```
@@ -0,0 +1,75 @@
+            docker stop ticketmon-backend || true
+            docker rm ticketmon-backend || true
+
+            echo "🔐 GHCR 로그인 중..."
+            # GHCR 로그인 (GitHub Actions 기본 토큰 사용)
+            # DOCKER_USERNAME 대신 github.actor를 사용하는 것이 일반적입니다.
+            echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+
+            echo "⬇️ 최신 Docker 이미지 Pull 중..."
+            # DOCKER_IMAGE_REPO secret에는 '조직명/레포지토리명' 전체가 들어가야 합니다 (예: 'aibe-3team/aibe1-finalproject-team03_be-aws-migration-test')
+            docker pull ghcr.io/${{ secrets.DOCKER_IMAGE_REPO }}:latest # 수정된 이미지 경로 사용
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)